tests/unit/wc-payment-api/test-class-wc-payments-api-client.php

```diff
@@ -1304,7 +1304,10 @@ private function create_woocommerce_default_pages(): array {
 	 *
 	 * @dataProvider data_determine_suggested_product_type
 	 */
-	public function test_determine_suggested_product_type( $order_items, $expected_product_type ) {
+	public function test_determine_suggested_product_type( $order_items, $expected_product_type, $feature_flag_enabled = true ) {
+		// Set the feature flag option.
+		update_option( WC_Payments_Features::DISPUTE_ADDITIONAL_EVIDENCE_TYPES, $feature_flag_enabled ? '1' : '0' );
```

Comment:
The "feature_flag_enabled" name feels a bit generic. What do you think of renaming the argument `$evidence_types_flag_enabled`? (or something similarly more specific)